modules/migrating-to-virt.adoc

```diff
@@ -6,20 +6,24 @@
 
-.Procedure
+* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator_mtv#installing-the-operator_mtv[Installing the {mtv-short} Operator]: Before you can use the toolkit, you must install the {mtv-short} Operator on your {product-title} cluster. The Operator provides all the necessary components for migration.
 
-* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vmware#adding-source-provider_vmware[Migrate virtual machines from {vmw-first}].
-* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-osp_ostack#adding-source-provider_ostack[Migrate virtual machines from {rh-openstack-first}].
-* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-rhv_rhv#adding-source-provider_rhv[Migrate virtual machines from Red Hat Virtualization].
-* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-virt_cnv#adding-source-provider_cnv[Migrate virtual machines from {VirtProductName}].
-* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-ova_ova#adding-source-provider_ova[Migrate virtual machines from OVA files created by {vmw-full}].
+* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index%23creating-a-migration-plan[Creating a migration plan]: A migration plan defines the VMs to be migrated, the source and destination environments, and other migration settings. This plan helps to organize and automate the migration process.
+
+* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index%23creating-a-network-map#about-network-maps_vmware[Mapping networks] and link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index%23creating-a-network-map#about-storage-maps_vmware[mapping storage]: {mtv-short} helps you to map networks and storage from your source environment (for example, vSphere or RHV) to the {product-title} environment. This ensures that the migrated VMs have the correct network and storage configurations.
```

Comment:
These three links (Creating a migration plan, Mapping networks, and mapping storage) don't work for me; they all point to the top of the page as if the links were all to https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.8/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index 

@anarnold97 Do the links work on your end?